sdk/purview/azure-purview-scanning/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0b2 (2021-09-24)
```

Comment:
see [this](https://github.com/Azure/azure-sdk-for-python/pull/20822#discussion_r717695891) comment